products/alinux2/profiles/cis.profile

```diff
@@ -0,0 +1,20 @@
+documentation_complete: true
+
+metadata:
```

Comment:
We would like to have note who is the maintainer of a profile. Therefore, we use the "SMEs" section in profile metadata.
You can see an example here:
https://github.com/ComplianceAsCode/content/blob/master/products/rhel8/profiles/cis.profile
Could you add this section with at least one github handle?